services/mail.rb

```diff
@@ -19,6 +19,10 @@ def receive_validate(errors)
         errors[:addresses] = "Must be a comma-separated string"
         return false
       end
+      if address.strip !~ /@/
+        errors[:addresses] = "Not a valid email address"
```

Comment:
It would be helpful to point out to the user which of the addresses they entered is not valid. i.e. change the error message to `"Not a valid email address: #{address}"`